tests/integration/clone.rs

```diff
@@ -9,21 +9,22 @@ use std::process::Command;
 
 #[test]
 fn test_toprepo_clone() {
-    let from_dir = git_toprepo_testtools::test_util::MaybePermanentTempDir::new_with_prefix(
-        "git_toprepo-test_toprepo_clone-from-",
+    let base_dir = git_toprepo_testtools::test_util::MaybePermanentTempDir::new_with_prefix(
+        "git_toprepo-test_toprepo_clone",
```

Comment:
Either add a trailing hyphen or let `new_with_prefix` (or `new_with_test_name`) set `git_toprepo-{test_name}-`.
```suggestion
        "git_toprepo-test_toprepo_clone-",
```